hip_data_tools/aws/s3.py

```diff
@@ -122,7 +124,8 @@ def move_recursive_to_different_bucket(self, source_dir, destination_bucket_name
         for obj in source_bucket.objects.filter(Prefix=source_dir):
             if not obj.key.endswith(file_suffix_filter):
                 new_key = "{destination_dir_without_bucket_name}/{destination_file_name}".format(
```

Comment:
do you need to sanitise this key? Maybe checking for double slashes?

Reply:
out of scope of this PR